app/build.gradle

```diff
@@ -80,6 +77,10 @@ android {
             includeAndroidResources true
         }
     }
+    lint {
```

Comment:
why are we changing the linting?

Reply:
these were all auto changes made when I updated the Gradle package

Reply:
but I have changed it back (i think, I'll double check)